### DISCUSSION TITLE
Less-verbose option for project-config/apply

### DISCUSSION THREAD

**Discussion by alexgit2k** [craftcms/cms#12568] [Ideas] [answered]
`project-config/apply` produces a lot of output when updates/installations are required, which makes it quite unclear in CI/CD-environments. For example, we are having over 5000 lines output.

Therefore I suggest a new option `--less-verbose` which for a successful run only outputs this:
> Uninstalling plugins ... done
> Applying changes from your project config files ... done
> Finished applying changes

**Comment by brandonkelly** [comment, marked as answer]
Added a `--quiet` option for Craft 4.4! (d046377df6c9e8d45b498e80965878d3353fe171)

**Reply by brandonkelly** [reply]
4.4 is out!
